cargo-typify/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "cargo-typify"
+version = "0.0.11-dev"
+edition = "2021"
+
+[dependencies]
+typify = { version = "0.0.11-dev", path = "../typify" }
+
+clap = { version = "4.1.6", features = ["derive"] }
```

Comment:
Pulled in `clap` for CLI Argument parsing

https://docs.rs/clap/latest/clap/index.html

Reply:
Yes. Good!